include/verilated.mk.in

```diff
@@ -271,10 +271,10 @@ ifneq ($(VM_DEFAULT_RULES),0)
 %.o: %.cpp
 	$(OBJCACHE) $(CXX) $(OPT_FAST) $(CXXFLAGS) $(CPPFLAGS) -c -o $@ $<
 
-$(VK_OBJS_FAST): %.o: %.cpp $(VK_PCH_H).fast.gch
+$(VK_OBJS_FAST): %.o: %.cpp $(VK_PCH_H).fast$(CFG_GCH_IF_CLANG)
```

Comment:
The dependency still needs .gch, the problem is the $< below needs to remove the .gch, so you can't use `$<`.